README.md

```diff
@@ -45,14 +50,16 @@ The top level of the configuration file looks like this:
       },
       "scheme": "https",
       "__snapName": "myUniqueSnapName",
-      "__snapType": "__GET"
+      "__snapType": "__REST""
```

Comment:
Looks like there’s an extra ending quote here!

Reply:
thank you :)